src/docfx/serve/lsp/DidChangeWatchedFilesHandler.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Workspace;
+
+namespace Microsoft.Docs.Build
+{
+    internal class DidChangeWatchedFilesHandler : IDidChangeWatchedFilesHandler
+    {
+        private readonly LanguageServerBuilder _languageServerBuilder;
+        private readonly LanguageServerPackage _package;
+
+        private readonly Container<FileSystemWatcher> _watcher = new Container<FileSystemWatcher>(
+                new FileSystemWatcher() { GlobPattern = "**/*", Kind = WatchKind.Create | WatchKind.Delete });
```

Comment:
>Create [](start = 81, length = 6)

add `WatchKind.Change`?

Reply:
yes, and `WatchKind.Change` should only update that file timestamp (not the whole directory stamp)